<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetConstantValue</class>
 <widget class="QWidget" name="WidgetConstantValue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Constant values</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="ConstantValueTable" name="tblValues"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Radionuclide related</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Symbol</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbSymbols"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="DischargeItemTable" name="tblRnValues"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DischargeItemTable</class>
   <extends>QTableWidget</extends>
   <header>dischargeitemtable.h</header>
  </customwidget>
  <customwidget>
   <class>ConstantValueTable</class>
   <extends>QTableWidget</extends>
   <header>constantvaluetable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
